inference-engine/ie_bridges/java/cpp/blob.cpp

```diff
@@ -104,6 +104,49 @@ JNIEXPORT jlong JNICALL Java_org_intel_openvino_Blob_BlobFloat(JNIEnv *env, jobj
     return 0;
 }
 
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_Blob_wrapMatToBlob(JNIEnv *env, jobject obj, jlong tensorDescAddr, jlong matDataAddr)
+{
+    static const char method_name[] = "wrapMatToBlob";
+    try
+    {
+        TensorDesc *tDesc = (TensorDesc *)tensorDescAddr;
+
+        Blob::Ptr *blob = new Blob::Ptr();
+        
+
+        Precision precision = tDesc->getPrecision();
+        std::vector<size_t> dims = tDesc->getDims();
+
+        size_t data_size = 1;
+        for (int i = 0; i < dims.size(); i++)
+            data_size *= dims[i];
+
+        if (precision == Precision::FP32) {
+            float *data = (float *) matDataAddr;
+            *blob = make_shared_blob<float>((*tDesc), data);
+
+            return (jlong)blob;
+        } else if (precision == Precision::U8) {
+            uint8_t *data = (uint8_t *) matDataAddr;
+            *blob = make_shared_blob<uint8_t>((*tDesc), data);
+
+            return (jlong)blob;
+        }
```

Comment:
Please throw exception for unsupported precision

Reply:
We checks precision value in TensorDesc constructor

Reply:
I see but we have no wrappers for I32 precision, in example. That means we need to tell user about it.

BTW, it'd be nice to have other precisions here as well.